lib/timeduration.test.js

```diff
@@ -7,61 +7,61 @@ import { t } from './withT'
     'now', // test name
     new Date(2018, 0, 11), // "now"
     new Date(2018, 0, 11), // date to compare now with
-    '0s' // expected return string
+    'vor 0 s' // expected return string
   ],
   [
     'minutes',
     new Date(2018, 0, 11, 0, 10, 0),
     new Date(2018, 0, 11, 0, 0, 0),
-    '10m'
+    'vor 10 m'
   ],
   [
     'hour',
     new Date(2018, 0, 11, 1, 0, 0),
     new Date(2018, 0, 11, 0, 0, 0),
-    '1h'
+    'vor 1 h'
   ],
   [
     'days',
     new Date(2018, 0, 13, 0, 0, 0),
     new Date(2018, 0, 11, 0, 0, 0),
-    '2t'
+    'vor 2 d'
```

Comment:
Would probably give these translation strings a go remove denglish, so:

"vor 2 d" becomes "vor 2 T"
"vor 1 w" becomes "vor 1 W"
"vor 1 y" becomes "vor 1 J"

While "s", "m" and "h" are :+1:

Reply:
Dani Meyer ultimately decided for English all the way